src/android/app/src/main/res/values/strings.xml

```diff
@@ -359,12 +359,25 @@
     <string name="emulation_open_settings">Open Settings</string>
     <string name="emulation_open_cheats">Open Cheats</string>
     <string name="emulation_switch_screen_layout">Landscape Screen Layout</string>
+    <string name="emulation_switch_portrait_layout">Portrait Screen Layout</string>
     <string name="emulation_screen_layout_landscape">Default</string>
     <string name="emulation_screen_layout_portrait">Portrait</string>
     <string name="emulation_screen_layout_single">Single Screen</string>
     <string name="emulation_screen_layout_sidebyside">Side by Side Screens</string>
     <string name="emulation_screen_layout_hybrid">Hybrid Screens</string>
-    <string name="emulation_cycle_landscape_layouts">Cycle Landscape Layouts</string>
+    <string name="emulation_screen_layout_above_below">Above and Below</string>
```

Comment:
I'm not sure about this layout name, I would suggest changing it to something else.
How about "Natural", referring to how it is the natural layout of the 3DS? Open to suggestions with this.

Reply:
Yeah, Above and Below is clunky. I was just trying to build off of side by side. But Top and Bottom isn't quite right either, and Vertical is confusable with portrait. Natural is maybe the best I've heard though it's not quite as descriptive (and might not make as much sense in translation). Maybe we ask the discord for other ideas?

Reply:
Okay, latest commit refactors this. 

I think we should change "Default" to "Large Screen (Default)" too. Otherwise I think Default/Original overlap too much. I haven't done that in this commit, but let me know if you agree.

Reply:
I think it's fine as it is now, changing the default landscape layout name would also necessitate changing the default portrait layout name despite there only being one not counting the custom layout.

Perhaps it can be revisited if we end up with more portrait layouts in the future.